src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -351,6 +351,7 @@ public void Properties_AddItemToDictionary_ItemPresent()
             }
         }
 
+        [ActiveIssue(28749)]
```

Comment:
Since the test failures are just on Linux, can't we restrict this ActiveIssue to TestPlatforms.AnyUnix ?

Reply:
It also fails on Mac with `System.Net.Sockets.SocketException : No route to host`. Could be similar root cause.

Besides, the passing Windows run implies behavior different (not necessary correct).

Since we will eventually resolve this issue, (modify the test/make product change), I don't think it's necessary to restrict the test right now.